site/AntDesign.Docs/Demos/Components/Statistic/demo/Separator.razor

```diff
@@ -0,0 +1,4 @@
+<Row Type="flex">
+    <Statistic Title="Number" Value="123456" GroupSeparator=" " Style="text-align: center;"></Statistic>
+    <Statistic Title="Decimal" Value="50.5" DecimalSeparator=" " Style="margin: 0 32px; text-align: center;"></Statistic>

```

Comment:
Why did you set the separators as a space?

Reply:
https://learn.microsoft.com/pl-pl/dotnet/api/system.globalization.numberformatinfo.numberdecimalseparator?view=net-7.0
Depending on the culture, we have different separators. For me it is a space in Poland/Europe. You can think about embedding the default culture value in the component if there is no separator.

Reply:
Oh, I see. Therefor we maybe can use ` LocaleProvider.CurrentLocale.CurrentCulture.NumberFormat.NumberDecimalSeparator` and `LocaleProvider.CurrentLocale.CurrentCulture.NumberFormat.NumberGroupSeparator` as default value.

Reply:
@ElderJames 
This can be done in the second PR. What with that? I think the documentation should be updated with an additional example.

Reply:
That's fine.